Library/Formula/validns.rb

```diff
@@ -0,0 +1,30 @@
+class Validns < Formula
+  homepage "http://www.validns.net/"
+  url "http://www.validns.net/download/validns-0.8.tar.gz"
+  sha256 "df2db0eaa998a0411ff4c1c4e417eb82d32aec4835f92f45f26c66c8d1d5bd22"
+
+  depends_on 'judy'
+
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!